libs/s25main/gameData/GameConsts.h

```diff
@@ -20,6 +20,10 @@ constexpr GameSpeed referenceSpeed = GameSpeed::Normal;
 
 /// Reichweite der Bergarbeiter
 constexpr unsigned MINER_RADIUS = 2;
+constexpr unsigned MINER_RADIUS_SETTLERSIV = 1u;
+
+/// maximum quantity for ores
+constexpr unsigned MINER_MAX_QUANTITY = 7u;
```

Comment:
This looks so arbitrary here. Especially as the documentation in the `Resource` class states
> Maximum number of resource types and amount is 15!

However it is correct in that the MapLoader uses `7` as the max value for ores, by design of the S2 map format, and also for water (by choice), while it uses 4 for fish.

We should probably also even check/assert this during loading.

BTW: The comment is a hint for an unfit name. How about `MAX_ORE_QUANTITY`? The current name sounds, like this is the maximum number of miners you can have and requires the comment to make it clear ;-)

It might still make sense to keep the comment but as something like "Maximum amount of the (mineable) ore resources`.  This adds 2 keywords: "mineable" and "resource" making it easier to know where this might be used.   
I hope you get my way to think about this.

Reply:
Maybe I also failed a bit to correctly distinguish, as I've never seen a well as a mine - which makes the name ```MINER_RADIUS_SETTLERSIV``` also wrong.